storage/storagepb/kv.proto

```diff
@@ -35,6 +35,10 @@ message Event {
   // a delete/expire event contains the previous
   // key-value
   KeyValue kv = 2;
-  // watchID is the ID of watching this event is sent to.
-  int64 watchID = 3;
+}
+
+message WatchResponse {
```

Comment:
No. Do not import that type. Just create WatchResponse in watcher.go as what I have did in my original commit. Do not use proto type unless you are going to marshal the exact struct.